CMakeLists.txt

```diff
@@ -840,6 +840,8 @@ set(DAEMON_FILES
         daemon/main.h
         daemon/signals.c
         daemon/signals.h
+        daemon/service.c
```

Comment:
Daemon is already overloaded with a lot of stuff. What do you think about creating a new top-level dir for this?

Reply:
Maybe later. I don't think we need to separate this code in advance.